addons/mail/static/src/core/common/message.xml

```diff
@@ -96,7 +96,17 @@
                                                         <button t-if="message.type === 'email'" class="o-mail-Message-originalEmailText d-block float-end p-1 mb-1 btn btn-link opacity-75 fst-italic" t-att-class="env.inChatWindow and isAlignedRight ? 'me-3' : 'ms-3'" t-esc="originalEmailText" t-on-click="() => this.state.originalEmail = !this.state.originalEmail"/>
                                                         <em t-if="message.subject and !message.isSubjectSimilarToOriginThreadName and !message.isSubjectDefault" class="mb-1 me-2">Subject: <t t-esc="message.subject"/></em>
                                                         <div t-if="message.type === 'email'" t-ref="shadowBody"/>
+                                                        <t t-elif="message.showTranslation" t-out="message.translationValue"/>
                                                         <t t-elif="message.body" t-out="message.body"/>
+                                                        <small class="fst-italic text-muted">
+                                                            <p t-if="message.showTranslation">
+                                                                (Translated from: <t t-if="message.translationSource" t-out="message.translationSource"/>)
+                                                            </p>
+                                                            <p t-if="message.translationErrors">
+                                                                <i class="text-danger fa fa-warning" role="img" title="Translation Failure" aria-label="Translation Failure"/>
```

Comment:
I don't really know, but assuming it is read by the TTS, I can only imagine how bad of a user experience it would be to hear “Translation Failure” three times in a row. And assuming it isn't, then those attributes are just plain useless 😺

Reply:
`tittle` adds a small popover effect when mouse-hovering. This can be quite annoying, as it hides content below.
More is not necessarily better, so use `title` cautiously :)